tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+```
+This command grants a `pubsub.publisher` role to the service account that was created previously.
+A few minutes after this command finishes you should see the warning icon removed from the export sink in the [Log Exports Console](https://console.cloud.google.com/logs/exports).
+
+## Deploying the code
+At this stage we have a export log that pushes all audit events into the topic we created.
+To complete the application we need some code to recieve these events, filter them and use the `Slack` API to post messages on the desired `Slack` channel.
+One way to do so that uses a fully manaaged, auoscalable, serveless comupting service is by using [Google CloudFunctions](https://cloud.google.com/functions/).
+This service will deploy and scale our code as much as required for optmial performance.
+
+`Google CloudFunctions` use [Node.js](https://nodejs.org/en/) as a runtime platform which means our code will be written using Javascript.
+
+Before we can make a `Google CloudFunctions` deployment we must create a `Google Cloud Storage` bucket for staging and enable the `Google CloudFunctions API` for our project.
+
+### 1) Create a staging bucket
+Run the following command:
+```
+$ gsutil mb -p <YOUR_PROJECT_ID> gs://<YOUR_PROJECT_ID>-gcp-alert-service
+```
+This command will create a new bucket for our deployment to stage in.
+
+### 2) Enable `Google CloudFunction API`
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta service-management enable cloudfunctions.googleapis.com
+```
+This command might take a few minutes. After it completes the `Google CloudFunctions` API will be enabled for your project.
+
+### 3) Writing the code
+To deploy into `Google CloudFunctions` we will use two files:
+- A `package.json` file that describes the module, dependencies etc'
+- An `index.js` file that exports the code to be run.
+
+### 4) Create a `package.json` file
+Create a file named `package.json`, copy the content from [this link](./package.json) into the file and save it locally.
+
+### 5) Create an `index.js` file
+Create a file named `index.js`, copy the content from [this link](./index.js) into the file and save it locally.
+
+### 6) Deploy the code
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta functions deploy gcp-alert-service --stage-bucket <YOUR_PROJECT_ID>-gcp-alert-service --trigger-topic gcp-alert-service --entry-point=pubsubLogSink --region=us-central1
+```
+This command will take a few minutes. 
+When complete you will can see the deployed function [here](https://console.cloud.google.com/functions/list).
+
+## Configuration
+Altough the code is deployed and events are being pushed - you will not yet recieve any `Slack` notifications. For that to happen the application must be configured.
```

Comment:
Typo and punctuation:
`Although` the code is deployed and events are being pushed`,` you will not yet `receive` any \`Slack\` notifications.